lib/modules/manager/crossplane/readme.md

```diff
@@ -0,0 +1,40 @@
+The `crossplane` manager has no `fileMatch` default patterns, so it won't match any files until you configure it with a pattern.
+This is because there is no commonly accepted file/directory naming convention for crossplane YAML files and we don't want to check every single `*.yaml` file in repositories just in case any of them have Crossplane Packages definitions: Configurations, Providers, Functions.
+
+If most `.yaml` files in your repository are Crossplane ones, then you could add this to your config:
+
+```json
+{
+  "crossplane": {
+    "fileMatch": ["\\.yaml$"]
+  }
+}
+```
```

Comment:
I don't know if "If most `.yaml files are Crossplane files" is good enough. Let's say you use GitHub Actions in your repository, those use `.yaml` as config language. When you set the `fileMatch` for your Crossplane files, I supect Renovate will mess up the GitHub Actions `.yaml` files when it updates?

I suspect that only when _all_ `.yaml` files are Crossplane files it's a decent idea to set the `fileMatch` like in the example?